ticktock-runtime/src/main/java/dev/zacsweers/ticktock/runtime/TzdbZoneDataProvider.java

```diff
@@ -41,7 +41,12 @@
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-public final class TzdbZoneProvider implements ZoneIdsProvider, ZoneDataProvider {
+/**
+ * A tzdb.dat-based {@link ZoneDataProvider}. This loads individual zone rules by zone ID from a
+ * {@link ZoneDataLoader}. The `tzdb.dat` file is expected to be found at a "known" path:
+ * {@code j$/time/zone/tzdb.dat}.
+ */
+public final class TzdbZoneDataProvider implements ZoneIdsProvider, ZoneDataProvider {
```

Comment:
I left out the `Data` because it's both a `ZoneData` and a `ZoneIds` provider but this is also fine.

Reply:
Yeah I thought about that but `TzdbZoneDataAndIdsProvider` field a little much :P